packages/components/src/components/combobox/shadow.tsx

```diff
@@ -542,6 +542,7 @@ export class KolCombobox implements ComboboxAPI {
 	@Watch('_suggestions')
 	public validateSuggestions(value?: SuggestionsPropType): void {
 		this.controller.validateSuggestions(value);
+		this._filteredSuggestions = value || [];
```

Comment:
Die Lösung sieht soweit richtig aus.

Brauchen wir den Array-Fallback (`|| []`) hier tatsächlich? `_filteredSuggestions` ist so definiert, dass es auch `undefined` sein darf.
In `componentWillLoad` setzen wir die Eigenschaft ebenfalls ungeprüft auf `this.state._suggestions`, ohne "Fallback".

Ich würde das Verhalten von componentWillLoad und Watcher lieber einheitlich halten.